Apps/AdminWebClient/test/unit/Services.Test/InactiveUserServiceTests.cs

```diff
@@ -0,0 +1,605 @@
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//-------------------------------------------------------------------------
+namespace HealthGateway.AdminWebClientTests.Services.Test
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
+    using HealthGateway.Admin.Server.Models;
+    using HealthGateway.Admin.Server.Services;
+    using HealthGateway.Common.AccessManagement.Administration;
+    using HealthGateway.Common.AccessManagement.Administration.Models;
+    using HealthGateway.Common.AccessManagement.Authentication;
+    using HealthGateway.Common.AccessManagement.Authentication.Models;
+    using HealthGateway.Common.Constants;
+    using HealthGateway.Common.Models;
+    using HealthGateway.Database.Constants;
+    using HealthGateway.Database.Delegates;
+    using HealthGateway.Database.Models;
+    using HealthGateway.Database.Wrapper;
+    using Microsoft.Extensions.Configuration;
+    using Microsoft.Extensions.Logging;
+    using Moq;
+    using Xunit;
+
+    /// <summary>
+    /// InactiveUserService's Unit Tests.
+    /// </summary>
+    public class InactiveUserServiceTests
+    {
+        private readonly IConfiguration configuration;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="InactiveUserServiceTests"/> class.
+        /// </summary>
+        public InactiveUserServiceTests()
+        {
+            this.configuration = GetIConfigurationRoot();
+        }
+
+        /// <summary>
+        /// GetInactiveUsers given inactive users in DB, Admin and Support users.
+        /// </summary>
+        [Fact]
+        public void ShouldGetInactiveUsers()
+        {
+            int expectedInactiveUserCount = 5;
+
+            // Arrange
+            Mock<IAuthenticationDelegate> authenticationDelegateMock = new();
+            authenticationDelegateMock.Setup(s => s.AuthenticateAsUser(It.IsAny<Uri>(), It.IsAny<ClientCredentialsTokenRequest>(), It.IsAny<bool>())).Returns(GetJwtModel());
+
+            DBResult<IEnumerable<AdminUserProfile>> activeProfileResult = new()
+            {
+                Status = DBStatusCode.Read,
+                Payload = new List<AdminUserProfile>()
+                {
+                    new AdminUserProfile
+                    {
+                        Username = "username1",
+                    },
+                    new AdminUserProfile
+                    {
+                        Username = "username2",
+                    },
+
+                    // Example: when 0 inactive days is entered, user will be both active and inactive
+                    new AdminUserProfile
+                    {
+                        Username = "username7",
+                    },
```

Comment:
I don't understand why `"username7"` is being returned as both active and inactive in these mocks when the test is simulating a call for users active in the last 10 days.